### PR TITLE
Edit people find

### LINKED ISSUE #141: Change people find command
Support searching for people's name, phone, email, tags. 
Add a "Debt" and "Loan" tag to people when owe/lend command is executed so that user can search for people with debts or loans.

### PR DESCRIPTION
**Pull request by CheyanneSim**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/people/PeopleFindCommandTest.java [hunk 1]
```diff
@@ -29,10 +29,10 @@
 
     @Test
     public void equals() {
-        NameContainsKeywordsPredicate firstPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameContainsKeywordsPredicate secondPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("second"));
+        PeopleNamePredicate firstPredicate =
```
Comment (gracegrape) [inline src/test/java/seedu/address/logic/commands/people/PeopleFindCommandTest.java:32]: For the predicates, will `PeopleNameContainsKeywordsPredicate` sound better, or is it too long haha. Cause `PeopleNamePredicate `might be a little unclear?

**Review by gracegrape — COMMENTED** [review]
Thank you for improving on people find! You code actually looks really good!

**Review by joel-lim — COMMENTED** [review]
Looks great! Thanks for upgrading our find command

**Review by syinyichen — COMMENTED** [review]
Looks good! Thanks for the work!

**Pull request merged**